internal/job/executor.go

```diff
@@ -363,6 +364,31 @@ func (e *Executor) runUnwrappedHook(ctx context.Context, hookName string, hookCf
 	return e.shell.RunWithEnv(ctx, environ, hookCfg.Path)
 }
 
+func logOpenedHookInfo(l shell.Logger, debug bool, hookName, path string) {
+	switch {
+	case runtime.GOOS == "linux":
+		procPath, err := file.OpenedBy(l, debug, path)
+		if err != nil {
+			l.Errorf("The %s hook failed to run because and we could not find the process that had it opened", hookName)
+		} else {
+			l.Errorf("The %s hook failed to run because it was open by %s", hookName, procPath)
```

Comment:
```suggestion
			l.Errorf("The %s hook failed to run the %s process has the hook file open", hookName, procPath)
```

Reply:
maybe something like `the %s process (PID: %d)` might be good? but would probably require a fair bit of plumbing so it's def not a blocker